helm/thingsboard/values.yaml

```diff
@@ -93,6 +93,8 @@ node:
   nodeSelector: {}
   tolerations: []
   affinity: {}
+  ruleChain:
+    configmapName:
```

Comment:
What about setting current default rule chain?  this way, we avoid to search for variables with unknown/hidden default values.
have a look at https://github.com/midokura/thingsboard-private/pull/862/files

Reply:
I did not do it because it is not needed here. The change is less intrusive in this way and has one more advantage:

If we add second copy of the same rule chain here, then we would have to maintain it and make sure it always matches the one from 'application/src/main/data' folder, right?

Reply:
makes sense. I have just realized that TBu can be deployed in so many ways, helm chart is just one of them!
let's keep this way so. thanks!